devel/glasskube/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+version             0.16.0
+github.setup        glasskube glasskube $version v
+categories          devel
+maintainers         openmaintainer {esafak @esafak}
+description         Missing Package Manager for Kubernetes
+long_description    Deploy, configure and update Kubernetes packages 20x faster than with Helm
+license             Apache-2
+homepage            https://glasskube.dev/
+
+# Checksums for verification
+checksums           rmd160 dcca396ad2d2b94eae6bad5d851f86271268ff84 \
+                    sha256 c65577b004b0a9d20c27e78c44be415960fe93e34995dba9858db975db83f404 \
+                    size   773135
+
+# Dependencies for build and runtime
+depends_build       port:go port:nodejs22 port:npm10
```

Comment:
Are there pseudo-ports I can use instead of specific versions?

Reply:
I haven't tried but it seems like the "npm" PortGroup could help here. Just check the available PortGroups under "_resources/,,,,", "grep" the ports tree, or check the guide.